src/main/java/com/minecolonies/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -36,8 +38,8 @@
     private static final int        NODE_DISTANCE             = 7;
     /**
      * Return to chest after 3 stacks
-     */
-    private static final int        MAX_BLOCKS_MINED          = 64 * 3;
+     *///todo change back to 64*3
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)